tests/tests/test_connect.py

```diff
@@ -151,6 +151,12 @@ def test_connect(self, clean_mongo, tenant_id=None):
                     "body": b"sh-5.0$ ",
                 }
 
+        for i in range(15):
+            obj = api_mgmt.get_device(dev.id)
+            if obj.status != 'connected':
```

Comment:
```suggestion
            if obj.status == 'connected':
```

Reply:
@alfrunes ... why?

Reply:
Shouldn't we break the loop once the device has connected?

Reply:
here we are waiting for the disconnection of the device; the next step (a few lines below) is checking we get 404

Reply:
Ohh, I just assumed it was the other way around. Sorry...